generic/tests/cfg/ntttcp.cfg

```diff
@@ -14,6 +14,9 @@
     kill_vm = yes
     vms += " vm2"
     driver_name = "netkvm"
+    driver_verifier = ${driver_name}
+    Win2016, Win2019, Win8..1, Win2012..r2:
+        driver_verifier += " ndis"
```

Comment:
What about
```
driver_verifer = "${driver_name} ndis"
Win7, xxx...:
    driver_verifer = "${driver_name}"
```
Otherwise you might need to update it every time when we have new version released, right?

Reply:
There is no need to update every release. I think if the `driver_verifier` needs to be changed in the future, we only need to add a new verifier according to different windows guests.What do you think about that?

Reply:
@leidwang So if Windows has new version it will not append "ndis"?